packages/components/tests/dummy/app/templates/components/form/base-elements.hbs

```diff
@@ -50,6 +50,16 @@
         <code class="dummy-code">&lt;label&gt;</code>
         element.</p>
     </dd>
+    <dt>isRequired <code>boolean</code></dt>
+    <dd>
+      <p>It appends a badge to the label text to indicate user input is required.</p>
+      <p>Default: <span class="default">false</span></p>
```

Comment:
[issue] if you are leaving this up to the user to declare, then the default would be `null` instead of `false`, yes?

Reply:
Technically correct. But probably, to keep it simple, we can leave it as is? For the consumer what is important is to know is that it can be `true/false` (and in this case the default would be _falsy_, but if we use `null` or `falsy` may be confusing?)